src/UIKit/UIApplication.cs

```diff
@@ -128,6 +128,26 @@ public static UIContentSizeCategory ToEnum (NSString key)
 			// Do not compare to Medium, it is the fallback
 			return UIContentSizeCategory.Medium;
 		}
+
+		public static NSString FromEnum (UIContentSizeCategory category)
```

Comment:
`src/Security/Enums.cs` (end of file) has an example

I thought I added the documentation a while ago, I'll double check and update if required.